src/middlewares/authMiddleware.js

```
@@ -1,3 +1,27 @@
+
+// src/middlewares/authMiddleware.js
+const jwt = require('jsonwebtoken');
+
+const SECRET_KEY = 'your-secret-key'; // 실제 서비스에서는 환경변수로 관리하세요!
+
+function verifyToken(req, res, next) {
+  const authHeader = req.headers['authorization'];
+  const token = authHeader && authHeader.split(' ')[1]; // Bearer <token>
+
+  if (!token) {
+    return res.status(401).json({ error: '토큰이 필요합니다.' });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)